docs/source/reference/api-server/api-server-tunning.rst

```diff
@@ -0,0 +1,152 @@
+            -   memory: 16+
+            +   cpus: 16+
+            +   memory: 32+
+
+The following table shows the maximum concurrency for different resource configurations:
+
+.. list-table::
+   :widths: 1 1 3 2
+   :header-rows: 1
+
+   * - CPU
+     - Memory
+     - Max concurrency
+     - Suggested team size
+   * - 4
+     - 8Gi
+     - 8 Long requests, 11 Short requests
+     - 1~5 users
+   * - 16
+     - 32Gi
+     - 32 Long requests, 60 Short requests
+     - 1~15 users
+   * - 32
+     - 64Gi
+     - 64 Long requests, 145 Short requests
+     - 1~30 users
+   * - 64
+     - 128Gi
+     - 128 Long requests, 299 Short requests
+     - 10~50 users
+   * - 128
+     - 256Gi
+     - 256 Long requests, 589 Short requests
+     - 10~100 users
+
+Queuing requests and polling status asynchronously
+--------------------------------------------------
+
+There is no limit on the number of queued requests, i.e. despite increasing the allocated resources to improve the maximum concurrency, you can also submit requests with :ref:`async<async>` (``--async``) and poll the status asynchronously to avoid blocking. For example:
+
+.. code-block:: bash
+
+    # Submit 2000 jobs at once without blocking
+    for i in `seq 1 2000`; do
+        sky jobs launch -y --async job.yaml
+    done
+    # Poll the status of the jobs
+    watch -n 5 "sky jobs queue"
+
+The requests will be queued on the API server and be processed in submission order. If you find the status is not updated for a while, you can inspect the status of the submitted requests with:
+
+.. code-block:: console
+
+    $ sky api status
+    ID                                    User  Name             Created         Status
+    d9c95c7e-d248-4a7f-b72e-636511405357  alice sky.jobs.launch  a few secs ago  PENDING
+    767182fd-0202-4ae5-b2d7-ddfabea5c821  alice sky.jobs.launch  a few secs ago  PENDING
+    5667cff2-e953-4b80-9e5f-546cea83dc59  alice sky.jobs.launch  a few secs ago  RUNNING
+
+Check logs for a request
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+There should be some ``RUNNING`` requests that occupy the concurrency limit. Usually the ``RUNNING`` requests make progress and finally your requests will be processed, but if the ``RUNNING`` requests are stuck, you can inspect the request log with:
+
+.. code-block:: console
+
+    # Replace <request_id> with the actual request id from the ID column
+    $ sky api logs <request_id>
+
+Cancel a request
+^^^^^^^^^^^^^^^^
+
+If the request is stuck according to the log, e.g. retrying to launch VMs that is out of stock, you can cancel the request with:
+
+.. code-block:: bash
+
+    sky api cancel <requst_id>
+
+Avoid concurrent logs requests
+------------------------------
+
+If you run ``sky logs`` to tail the logs of a task, the log tailing will keep taking off the resources of the API server as long as the task being tailed is running. So concurrent log requests will occupy the concurrency and make other requests to be delayed.
```

Comment:
In this section, both 'task' and 'job' are used; can we keep one?

Reply:
Sure, let's stick to "task" which covers both `exec` and `jobs launch` I think